docs/cli/dotnet-install-script.md

```diff
@@ -0,0 +1,119 @@
+dotnet-install scripts reference
+================================
+
+## NAME
+dotnet-install.ps1 | dotnet-install.sh - script used to install the CLI tools and shared runtime
+
+## SYNOPSIS
+dotnet-install.sh [--channel] [--version]
+    [--install-dir] [--debug] [--no-path] 
+    [--shared-runtime]
+
+dotnet-install.ps1 [-Channel] [-Version]
+    [-InstallDir] [-Debug] [-NoPath] 
+    [-SharedRuntime]
+
+## DESCRIPTION
+The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. Their 
+main use case is to help with automation scenarios and non-admin installations. There are two scripts, one for PowerShell 
+that works on Windows and a bash script that works on Linux/OS X. They boht have the same behavior. Bash script also 
+"understands" PowerShell switches so you can use them across the board. 
+
+Installation scripts will download the ZIP/tarball from the CLI build drops and will proceed to install it in either the 
+specified location or in a different location is the `--install-dir` is specified. By default, the installation script 
+will download the entire CLI and install it; if you want to get just the shared runtime, you can specify the `--shared-runtime`
+argument. 
+
+By default, the script will add the install location to the $PATH for the current session. This can be overriden if the 
+`--no-path` argument is used. 
+
+The install scripts will not install dependencies that are needed for each of the component. That is up to the invoker 
+to do prior to installing bits via the script. 
+
+You can install a specific version using the `--version` argument. The version needs to be specified as 3-part version 
+(e.g. 1.0.0-13232). If omitted, it will default to the first global.json that contains the sdkVersion property and if 
+that is not present it will use Latest.
+
+## Options
+Options are different between script implementations. 
+
+### PowerShell
+`-Channel [CHANNEL]`
+
+Which channel (i.e. "future", "preview", "production") to install from. Defaults to "Production".
+
+`-Version [VERSION]`
+
+Which version of CLI to install; you need to specify the version as 3-part version (i.e. 1.0.0-13232). If omitted, it will default to the first global.json that contains the sdkVersion property; if that is not present it will use Latest. 	
```

Comment:
According to Microsoft Writing Style Guide, we shouldn't use i.e. or e.g. It should be that is or for example.